lib/tasks/combine_schemas.rake

```diff
@@ -30,7 +30,7 @@ end
 
 def sources_for_v2_schema(filename)
   Rake::FileList.new(
-    "formats/definitions.json",
+    "formats/{definitions,base_edition_links}.json",
```

Comment:
To generate a v2 `schema.json` file, information from both `formats/definitions.json` and `formats/base_edition_links.json` is required.